packages/commonwealth/client/scripts/controllers/app/webWallets/polkadot_web_wallet.ts

```diff
@@ -33,7 +29,7 @@ class PolkadotWebWalletController
   public readonly chain = ChainBase.Substrate;
 
   public get available() {
-    return isWeb3Injected;
```

Comment:
The underlying npm pkg uses this condition to check for availability but does not update the `isWeb3Injected` reliably
```
Object.keys(win.injectedWeb3).length !== 0
```